todoApp/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <title>ToDoリスト</title>
+    <script src="https://unpkg.com/vue@3/dist/vue.global.js"></script>
+    <script defer src="main.js"></script>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <h1>ToDoアプリ</h1>
+    <div id="todoList">
+      <h2>ToDo追加</h2>
+      <input v-model="newTodoText" />
+      <button @click="add(newTodoText)">追加</button>
+      <div class="error" v-if="noInput">ToDoを入力してください</div>
+
+      <h2>ToDo一覧</h2>
+      <table>
+        <tbody>
+          <tr v-for="todo in todoItems" :key="todo.id">
+            <td v-if="todo.isEditing"><input v-model="todo.text" /></td>
+            <td v-else>{{ todo.text }}</td>
+            <td>
+              <button @click="toggleEditState(todo)">
+                {{editButtonText(todo)}}
```

Comment:
こちらもスタイルと揃えると良さそうです。

```suggestion
                {{ editButtonText(todo) }}
```